cerberus/kubernetes/client.py

```diff
@@ -45,8 +46,11 @@ def list_pods(namespace):
     try:
         ret = cli.list_namespaced_pod(namespace, pretty=True)
     except ApiException as e:
-        logging.error("Exception when calling \
-                       CoreV1Api->list_namespaced_pod: %s\n" % e)
+        logging.error(
+            "Exception when calling \
+                       CoreV1Api->list_namespaced_pod: %s\n"
+            % e
```

Comment:
@paigerube14 The spacing here looks a little weird, thoughts?

Reply:
yes very weird. Every time I run the pre-commit run --all-files it edits back to that. Do you know what flake rule might be causing that. Trying to investigate